src/changelog/2024/02/device-instance-audit-logs.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Device Instance Audit Logging
+date: 2024-02-07 13:00:00.0
+authors: ["stephen-mclaughlin"]
+tags:
+    - changelog
+---
+
+With the introduction of the Device Auditlog last month, now we collect and display the logging events from the Node-RED instance running on your device.
+
+This brings closer parity with the logging you get from a hosted instance.
+
+Additional Events Recorded in the Device Auditlog include:
+
+- Context Key Deleted
+- Node-RED Settings Updated
+- Flows Reloaded
+- Flow Deployed
+- Nodes Installed
+- Nodes Removed
+- Library Item Saved
+
```

Comment:
```suggestion
With the recent introduction of the Device audit log, we've now updated the Device Agent to start logging events from the Node-RED instance running on your device.

This brings closer parity with the logging you get from a hosted instance.

The audit log events include details of the flows being updated, nodes being added or removed from the palette and other changes to the configuration.

Update to Device Agent v2.2 to start receiving the events.

```

Slight rewording. As this requires the user to manually upgrade to a specific version of the device agent, we need to call that out.